view/frontend/templates/breadcheckout/cartview.phtml

```diff
@@ -77,25 +81,43 @@
                     shippingAddress.block_key        = '<?php /* @noEscape */ echo $block->getBlockCode(); ?>';
                     shippingAddress.additional_data  = <?php /* @noEscape */ echo  $block->jsonEncode($block->setAdditionalData()); ?>;
 
+                    var taxEstimationUrl = '<?php /* @noEscape */ echo $block->getTaxEstimationUrl(); ?>';
+                    var shippingString = JSON.stringify(shippingAddress);
+                    var errorInfo;
+
                     $.ajax({
-                        url: '<?php /* @noEscape */ echo $block->getTaxEstimationUrl(); ?>',
-                        data: {shippingInfo: JSON.stringify(shippingAddress)},
+                        url: taxEstimationUrl,
+                        data: { shippingInfo: shippingString },
                         type: 'post'
                     }).done(function (response) {
                         try {
                             if (typeof response == 'object') {
```

Comment:
what happens if response is not an object?

Reply:
👍 good call, I'll add an else there to catch and record if that ever happens